.github/workflows/swift-toolchain.yml

```diff
@@ -197,9 +195,6 @@ jobs:
             fi
           fi
 
-          echo windows_build_runner=${{ vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
```

Comment:
Could you write a json list in the output variable and use GHA expressions to parse and expand it in `runs-on`?

Reply:
I will revert to the previous system before I properly send it for review if that's preferred. I believe it's better to keep the VM size in this file instead of using a GHA variable but I'm fine with keeping the variable if it's preferred, it's going to make the logic more complicated because the runs-on has to include the run_id in the case of cirun but not in the case of hosted VM as it is the case at the moment.

Reply:
Ok, I have no strong views, but fyi @compnerd (I just realized he wasn't on this PR)

Reply:
The problem is that it makes it very difficult for someone to replicate this repository and contribute. There are others using the repository as well, so finding a way to make it possible to have others contribute without access or the need to replicate the full CI builders would be desirable.

Reply:
Oh thanks, that's useful insight.